configure/configure-ios.ts

```diff
@@ -19,7 +19,9 @@ const updateIos = async() => {
     await project.ios?.updateInfoPlist(appTarget.name, null, {
       NSCameraUsageDescription: 'This app needs Camera Permission to scan QR code.',
       NSFaceIDUsageDescription: 'This allows you to securely sign into "AbcPay Wallet".',
-      NSUserNotificationsUsageDescription: 'Notification may include alerts, sounds and icon badges. These can be configured in Settings.'
+      NSUserNotificationsUsageDescription: 'Notification may include alerts, sounds and icon badges. These can be configured in Settings.',
```

Comment:
Enable notification, location service and regularly check-in AbcPay wallet so businesses can pay your for your attention. Your identity is completely private.